packages/binding-http/test/http-server-test.ts

```diff
@@ -278,18 +367,18 @@ class HttpServerTest {
         testThing.properties.maintenanceNeeded.forms = [];
         testThing.actions.makeDrink.forms = [];
 
-        const td = testThing.getThingDescription();
+        testThing.getThingDescription();
```

Comment:
doesn't really matter but I think the call can be removed

Reply:
true! I'll remove it

Reply:
Fixed, 👍🏻